examples/tracing-grpc/Cargo.toml

```diff
@@ -19,7 +19,7 @@ opentelemetry_sdk = { path = "../../opentelemetry-sdk", features = ["rt-tokio"]
 opentelemetry-stdout = { workspace = true, features = ["trace"] }
 prost = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
-tonic = { workspace = true, features = ["server"] }
+tonic = { workspace = true, features = ["server", "codegen", "channel", "prost"] }
```

Comment:
Curious - Why isn't there CI build failure because of these missing feature dependencies?

Reply:
Mystery for me! I can confirm that this example is indeed built from CI and is having no errors!.

Reply:
do you really need "server" in release dependencies here?

Reply:
or may be move examples to other repo? so that examples do not need "server" which leads to "axum" which fails usage of tonic without "axum" (so tonic also not so great as it has no separate axum feature)

Reply:
Only examples pull in server feature and hence Axum, right? Since it is not part of published crates, is that a concern?

Reply:
i guess you are right.